src/Feature/Editor/Draw.re

```diff
@@ -79,6 +79,7 @@ let drawText = (~context, ~x, ~y, ~paint, text) =>
   );
 let text = drawText;
 
+let underlinePaint = Skia.Paint.make();
 let underline =
     (~context, ~buffer, ~leftVisibleColumn, ~color=Colors.black, r: Range.t) => {
```

Comment:
I would have put this inside the scope of `underline`, so it doesn't pollute the namespace and can't accidentally be mutated elsewhere, but that's a minor nit.

Reply:
Ah, you fixed this for me too (by switching to use `drawRect` with color) - thank you!